migration-from-4x.md

```diff
@@ -506,72 +575,63 @@ editor.windowManager.openUrl({
+For information about the new URL dialog configuration, see: [URL dialog]({{site.baseurl}}/ui-components/urldialog/).
 
 ## Plugins
 
-Each release of TinyMCE adds new features and functionality. Occasionally some features and functionalities are removed because a better option is added.
-Here are the details about the features and functionalities that were removed in TinyMCE 5.0.
+The following section covers the changed and removed plugin features for TinyMCE 5.
+
+* [Removed plugin settings](#removedpluginsettings)
+* [Changed plugin settings](#changedpluginsettings)
+* [Changed plugins](#changedplugins)
+* [Spellchecker plugin](#spellcheckerplugin)
+* [Table plugin](#tableplugin)
 
-### Removed features
+### Removed plugin settings
 
-Plugins with dialogs no longer have `height` or `width` settings for their dialogs. The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions.
+The `height` or `width` settings have been removed for plugin dialogs. The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions.
 
-The following plugins from TinyMCE 4.x do not require height or width options to be specified in TinyMCE 5.0:
+The following plugins from TinyMCE 4 do not require height or width options to be specified in TinyMCE 5:
 
 * [Code]({{site.baseurl}}/plugins/code/)
 * [Codesample]({{site.baseurl}}/plugins/codesample/)
 * [Preview]({{site.baseurl}}/plugins/preview/)
 * [Template]({{site.baseurl}}/plugins/template/)
 
-#### Changed plugin settings
+### Changed plugin settings
 
-| **TinyMCE 4.x** | **TinyMCE 5.0** |
-| [`autoresize_min_height`]({{site.url}}/docs-4x/plugins/autoresize/#autoresize_min_height) | [`min_height`]({{site.baseurl}}/configure/editor-appearance/#min_height) |
-| [`autoresize_max_height`]({{site.url}}/docs-4x/plugins/autoresize/#autoresize_max_height) | [`max_height`]({{site.baseurl}}/configure/editor-appearance/#max_height) |
-| [`textcolor_cols`]({{site.url}}/docs-4x/plugins/textcolor/#textcolor_cols) | [`color_cols`]({{site.baseurl}}/configure/content-appearance/#color_cols) |
-| [`textcolor_map`]({{site.url}}/docs-4x/plugins/textcolor/#textcolor_map) | [`color_map`]({{site.baseurl}}/configure/content-appearance/#color_map) |
+| **TinyMCE 4.x**                                                                           | **TinyMCE 5.0**                                                           |
```

Comment:
Should this be `TinyMCE 4` as I noticed `4.x` was changed to `4` in other places?

Reply:
Updated